src/ray/rpc/node_manager/node_manager_client.h

```diff
@@ -57,6 +57,53 @@ class NodeManagerClient {
   ClientCallManager &client_call_manager_;
 };
 
+/// Client used by workers for communicating with a node manager server.
+class NodeManagerWorkerClient
```

Comment:
How about just `NodeManagerClient`?

Reply:
The other one for cross-NodeManager communication is called that :/ or are you suggesting to combine?